modules/tutorials/pages/java-tutorial/install-couchbase-java-sdk.adoc

```diff
@@ -0,0 +1,158 @@
+= Set Up and Connect the Couchbase Java SDK
+:description: Learn how to configure the cluster connection and set up the Couchbase Java SDK to connect and interact with your student cluster.
+:page-pagination: full
+:page-topic-type: tutorial
+
+[abstract]
+{description}
+
+== Prerequisites
+
+Before continuing this tutorial, you must first:
+
+* xref:create-couchbase-cluster.adoc[Create a Capella Operational Cluster]
+* xref:buckets-scopes-and-collections.adoc[Implement the Data Model]
+* Install the Java Software Development Kit (version 8, 11, 17, or 21)
```

Comment:
Let's give a steer to people who have a choice about their environment, something along the lines of:

We recommend running the latest Java LTS version (i.e. at the time of writing JDK 21) with the highest patch version available.

Reply:
(I realise SDKman will also take care of this, but we want to be clear with recommendations.)